### PR TITLE
Provide a link to the geometry paper.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: doc/doxygen/headers/glossary.h [hunk 1]
```diff
@@ -1033,6 +1033,31 @@
...
+ * <dd>The "geometry paper" is a paper by L. Heltai, W. Bangerth, M. Kronbichler,
+ * and A. Mola, titled
+ * "Using exact geometry information in finite element computations", that
+ * describes how deal.II describes the geometry of domains. In particular,
+ * it discusses the algorithmic foundations on which the Manifold class
+ * is based, and what kind of information it needs to provide for mesh
+ * refinement, the computation of normal vectors, and the many other places
+ * where geometry enters into finite element computations.
+ *
+ * The paper is currently available on arXiv at https://arxiv.org/abs/1910.09824 .
```
Comment (tjhei) [inline doc/doxygen/headers/glossary.h]: do you want a link here?

File: doc/doxygen/headers/glossary.h [hunk 2]
```diff
@@ -1033,6 +1033,31 @@
...
+ * The full reference for this paper is as follows:
+ * @code{.bib}
+@misc{heltai2019using,
+    title={Using exact geometry information in finite element computations},
+    author={Luca Heltai and Wolfgang Bangerth and Martin Kronbichler and Andrea Mola},
+    year={2019},
+    eprint={1910.09824},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
```
Comment (tjhei) [inline doc/doxygen/headers/glossary.h]: no doi and no url?

File: doc/doxygen/headers/glossary.h (hunk 1 above)
Comment (bangerth) [reply doc/doxygen/headers/glossary.h]: doxygen recognizes this as the link.

File: doc/doxygen/headers/glossary.h (hunk 2 above)
Comment (bangerth) [reply doc/doxygen/headers/glossary.h]: Well, the URL is above already, and arXiv papers have no DOIs :-)

File: doc/doxygen/headers/glossary.h (hunk 1 above)
Comment (tjhei) [reply doc/doxygen/headers/glossary.h]: doh, of course. Somehow I thought this was html source.

File: doc/doxygen/headers/glossary.h (hunk 2 above)
Comment (tjhei) [reply doc/doxygen/headers/glossary.h]: I think it is silly that the arxiv citation window doesn't add the url into the bibtex. That would be useful for anybody copying this. Anyways, not important...

**Comment by bangerth** [comment]
I'll update the html in the bibtex once the paper is actually accepted/published.

**Pull request merged**